src/ckb-daemon/usb_linux.c

```diff
@@ -65,9 +65,9 @@ static char kbsyspath[DEV_MAX][FILENAME_MAX];
 /// If DEBUG_USB is set during compilation,
 /// the number of bytes sent and their representation are logged to the error channel.
 ///
-int os_usbsend(usbdevice* kb, const uchar* out_msg, int is_recv, const char* file, int line){
+int os_usbsend(usbdevice* kb, const uchar* out_msg, int is_recv, const char* file, int line) {
     int res;
-    if(kb->fwversion >= 0x120 && !is_recv){
+    if (kb->fwversion >= 0x120 && !is_recv) {
         struct usbdevfs_bulktransfer transfer;
```

Comment:
Thanks, a fine function, I didn't know this struct initializer.
I changed it, but I believe it may give conflicts with other branches from @tatokis.